scripts/supabase-test-setup.sh

```diff
@@ -9,11 +9,11 @@ git clone --no-checkout https://github.com/technologiestiftung/stadtpuls-supabas
   && git config advice.detachedHead false \
   && mkdir -p .git/info \
   && echo supabase-docker-compose >> .git/info/sparse-checkout \
-  && git checkout v4.0.0
+  && git checkout v4.1.0
 
 # Remove git folder to avoid changes within supabase folder
 # to be recognised as changes to a subrepo
-rm -rf ./.git
+rm -rf ../supabase/.git
```

Comment:
Actually It shouldn't have happened. I'm still not sure how it did. However with this change this shouldn't happen by accident anymore, as it can only delete a .git folder within a supabase folder.